app/Console/Commands/UserCreate.php

```diff
@@ -71,23 +87,52 @@ public function handle(DiscourseService $discourseService): void
             $this->error("Invalid parameters " . $validator->messages()->toJson());
         } else
         {
-            $user = User::create([
-                                     'name' => $name,
-                                     'email' => $email,
-                                     'password' => Hash::make($password),
-                                     'role' => Role::RESTARTER,
-                                     'recovery' => substr(bin2hex(openssl_random_pseudo_bytes(32)), 0, 24),
-                                     'recovery_expires' => strftime('%Y-%m-%d %X', time() + (24 * 60 * 60)),
-                                     'calendar_hash' => Str::random(15),
-                                     'username' => '',
-                                     'wiki_sync_status' => WikiSyncStatus::CreateAtLogin,
-                                     'language' => $language,
-                                     'repair_network' => $repair_network_id,
-                                 ]);
+            $userData = [
+                'name' => $name,
+                'email' => $email,
+                'password' => Hash::make($password),
+                'role' => $role,
+                'recovery' => substr(bin2hex(openssl_random_pseudo_bytes(32)), 0, 24),
+                'recovery_expires' => strftime('%Y-%m-%d %X', time() + (24 * 60 * 60)),
+                'calendar_hash' => Str::random(15),
+                'username' => '',
+                'wiki_sync_status' => WikiSyncStatus::CreateAtLogin,
+                'language' => $language,
+                'repair_network' => $repair_network_id,
+            ];
+
+            // Add consent data if provided
+            if (!empty($consent_past_data)) {
+                $userData['consent_past_data'] = $consent_past_data;
+            }
+            
+            if (!empty($consent_future_data)) {
+                $userData['consent_future_data'] = $consent_future_data;
+            }
+            
+            if (!empty($consent_gdpr)) {
+                $userData['consent_gdpr'] = $consent_gdpr;
+            }
```

Comment:
These values are either string dates or null, right? I think we can simplify it.

```suggestion
                'repair_network' => $repair_network_id,
                'consent_past_data' => $consent_past_data ?? '',
                'consent_future_data' => $consent_future_data ?? '',
                'consent_gdpr' => $consent_gdpr ?? '',
            ];
```

Reply:
Addressed in [c428255](https://github.com/iFixit/restarters/pull/15/commits/c4282554dc2afebfc3aa1dab2415c40d36480569)